Perlin Noise Cloud

edwinstraub opened the pull request:
Cloud generator function using perlin noise function.

edwinstraub committed:
Perlin Noise Cloud

meiamsome commented:
This cloud doesn't appear to use perlin noise as far as I can tell - is it correct? Do you have another name for it if it is?

meiamsome commented:
Okay, I renamed this to Straub cloud for now to merge. Let me know if you want it changed, though

edwinstraub commented:
Hi,
  Thanks for the response. I've changed the function a bit. Hope it works
better now.

Have a great day,

Edwin

On Fri, Sep 29, 2017 at 7:34 PM, Oliver Wright <notifications@github.com>
wrote:

> Okay, I renamed this to Straub cloud for now to merge. Let me know if you
> want it changed, though
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CodingTrain/CommunityClouds/pull/14#issuecomment-333175271>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AWWjMjXQcPOkRfbnZmtcM-7VZbkGI9zaks5snRwYgaJpZM4PorYL>
> .
>
